lessons/session_1/session_1_introduction.md

```diff
@@ -0,0 +1,42 @@
+---
+marp: true
+theme: uncover
+_class: invert
+paginate: true
+output: pdf_document
+---
+
+# ML Systems in Production
+# Session 1: Introduction
+Jesús A. Calderón
+February 2024
+
+---
+
+# Agenda
+
+**1.1 Overview of ML Systems**
+	
++ When to Use ML
++ ML in Production
++ ML vs traditional software
+	
+**1.2 Introduction to ML System Design**
+	
+ + Business and ML Objectives
+ + Requirements of Data-Driven Products: Reliability, Scalability, Maintainability, Adaptability
+ + Iterative Process
+ + Framing ML Problems 
+
+---
+
+# Agenda (cont.)
+
+**Assignment: Setting Up a Code Repository**
+
+ + Sign up for a GitHub Account
```

Comment:
As a part of our Certificates tech onboarding, we have already made sure learners all have GitHub accounts and have installed git locally.

Reply:
Removed.